pkgs/development/python-modules/pytest-responses/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, python3
+, fetchFromGitHub
+,
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

Comment:
```suggestion
buildPythonPackage rec {
```

Reply:
This is not an application right? The examples on the project docs look like a regular python module.

Reply:
No is not an application.